share/completions/nu.fish

```diff
@@ -0,0 +1,16 @@
+complete -c nu -s h -l help -d 'Display help and exit'
+complete -c nu -s v -l version -d 'Display version and exit'
+
+complete -c nu -l stdin -d 'Redirect stdin'
+complete -c nu -s l -l login -d 'Use login shell'
+complete -c nu -s i -l interactive -d 'Use interactive mode'
+complete -c nu -l testbin -d 'Run internal test binary and exit'
+complete -c nu -s c -l commands -d 'Run commands and exit'
+complete -c nu -l config -d 'Specify config'
+complete -c nu -l env-config -d 'Specify environment config'
+complete -c nu -l log-level -a 'error warn info debug trace' -d 'Specify log level'
```

Comment:
For options which take a fixed list of arguments (`log-level`, `log-target`) you want to  pass `-x` to suppress file completions.